### PR TITLE
Update org addresses using a rake task

### PR DESCRIPTION
**Pull request by holdenhinkle**
Resolves https://github.com/department-of-veterans-affairs/va.gov-team/issues/81548

## Summary
The ARM team needs to update each accredited organization's address. This rake task will handle this.

## Related issue(s)
- https://github.com/department-of-veterans-affairs/va.gov-team/issues/81548

## Testing done

- [ ] *New code is covered by unit tests*
- *Describe what the old behavior was prior to the change*
- *Describe the steps required to verify your changes are working as expected. Exclusively stating 'Specs run' is NOT acceptable as appropriate testing*
- *If this work is behind a flipper:*
  - *Tests need to be written for both the flipper on and flipper off scenarios. [Docs](https://depo-platform-documentation.scrollhelp.site/developer-docs/feature-toggles-guide#Featuretogglesguide-Backendexample).*
  - *What is the testing plan for rolling out the feature?*

## Screenshots
_Note: Optional_

## What areas of the site does it impact?
Fields in the veteran_organizations table that only the ARM team uses.

## Acceptance criteria
- [ ]  I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [ ]  No error nor warning in the console.
- [ ]  Events are being sent to the appropriate logging solution
- [ ]  Documentation has been updated (link to documentation)
- [ ]  No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [ ]  Feature/bug has a monitor built into Datadog or Grafana (if applicable)
- [ ]  If app impacted requires authentication, did you login to a local build and verify all authenticated routes work as expected
- [ ]  I added a screenshot of the developed feature

## Requested Feedback

(OPTIONAL)_What should the reviewers know in addition to the above. Is there anything specific you wish the reviewer to assist with. Do you have any concerns with this PR, why?_

### CODE REVIEW

**Review by cosu419 — DISMISSED** [review]
LGTM. Approved

File: modules/veteran/lib/tasks/update_vso_addresses.rake [hunk 1]
```diff
@@ -0,0 +1,910 @@
...
+
+namespace :veteran do
+  desc 'Update VSO (organization) Addresses'
+  task update_vso_addresses: :environment do
+    num_records_updated = 0
+    num_records_not_found = 0
+    num_records_errored = 0
+
+    ADDRESSES.each do |org|
+      record = Veteran::Service::Organization.find(org['POA'])
```
Comment (jvcAdHoc) [inline modules/veteran/lib/tasks/update_vso_addresses.rake:893]:
```suggestion
      record = Veteran::Service::Organization.find(org[:poa])
```

**Comment by jvcAdHoc** [comment]
A concern I have is around address validation. Right now the only organization addresses we'd serve to users are ones that have been validated. I don't know if all of these addresses are real and what that means on the front end with google maps links.

**Comment by holdenhinkle** [comment]
Added address validation.

To avoid LightHouse rate-limiting us, I parameterized the rake task and created constants that each contain an array of up to 20 organization/addresses to update. When running the rake task you can pass in a constant name. See the code comments for instructions.

**Review by jvcAdHoc — APPROVED** [review]

**Pull request merged**